procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -120,90 +120,11 @@
   {
    "cell_type": "markdown",
    "id": "014ff42a-1d20-4a15-a145-3a59306459e2",
-   "metadata": {},
-   "source": [
-    "### If the camera and/or archiver don't come up, the commands below should bring them up."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "398422df-64ad-4da9-80c5-df4e02626141",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "tmp=await latiss.rem.atoods.cmd_start.set_start()\n",
-    "print(tmp)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "728bcdae-3e1e-4651-9519-50ec1fde129b",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "await salobj.set_summary_state(latiss.rem.atoods, salobj.State.STANDBY)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "aa223e1a-5501-4489-ae3e-f2b8ba523153",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "await salobj.set_summary_state(latiss.rem.atoods, salobj.State.ENABLED)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "36387d0e-d027-4872-a313-c3bccf63af35",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# ATCamera didn't come up\n",
-    "tmp=await latiss.rem.atcamera.cmd_start.set_start(timeout=10)\n",
-    "print(tmp)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "34ce130f-e542-4d19-9aea-55aa04f929d6",
    "metadata": {
     "tags": []
    },
-   "outputs": [],
    "source": [
-    "await salobj.set_summary_state(latiss.rem.atcamera, salobj.State.STANDBY)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "b255cf42-0d89-4de9-ba51-debe4ff32ca5",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "await salobj.set_summary_state(latiss.rem.atcamera, salobj.State.ENABLED)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "0b014428-5475-4530-8c42-2b969c315bf6",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# All up now"
+    "If the camera and/or archiver don't come up, see section `Quick Troubleshooting` for support commands. "
```

Comment:
should probably replace archiver with atoods since the csc was (sort of) renamed.

Reply:
Completely right!